corehq/apps/reminders/models.py

```diff
@@ -1484,7 +1488,13 @@ def _recipient_lookup(self):
         elif handler.recipient == RECIPIENT_SURVEY_SAMPLE:
             return CommCareCaseGroup.get(handler.sample_id)
         elif handler.recipient == RECIPIENT_OWNER:
-            return get_wrapped_owner(get_owner_id(self.case))
+            owner_id = get_owner_id(self.case)
+
+            location = SQLLocation.by_location_id(owner_id)
+            if location:
+                return [location]
```

Comment:
not too familiar with this part of the code, but any reason you're wrapping it in a list (as opposed to just checking explicitly for the type inside `recipient_is_location`)? assume it's a list elsewhere so you would need to support both structures, but just seems a little odd. might be a little clearer if you pushed the value/list abstraction into `recipient_is_location` instead